src/woo_publications/publications/admin.py

```diff
@@ -448,6 +448,19 @@ class TopicAdmin(AdminAuditLogMixin, admin.ModelAdmin):
     # TODO: uncomment actions when we offer the data to GPP-zoeken
     # actions = [sync_to_index, remove_from_index]
 
+    def get_search_results(self, request, queryset, search_term):
+        queryset, use_distinct = super().get_search_results(
+            request, queryset, search_term
+        )
+        # Order the queryset of the PublicationAdmins autocomplete_field `onderwerpen`
+        if (
+            "autocomplete" in request.path
+            and request.GET.get("model_name") == "publication"
+            and request.GET.get("field_name") == "onderwerpen"
+        ):
+            queryset = queryset.order_by("officiele_titel")
+        return queryset, use_distinct
```

Comment:
I can't believe Django doesn't have a straight forward way for this... apparently it uses the queryset of the related model admin. Anyway, by looking in the Django source code I believe this approach to be much cleaner:

```suggestion
    def get_queryset(self, request: HttpRequest):
        qs = super().get_queryset(request)
        if request.path == reverse("admin:autocomplete"):
            qs = qs.order_by("officiele_titel")
        return qs
```